openid-sharedsignals-framework-1_0.md

```diff
@@ -1761,36 +1769,36 @@ Errors are signaled with HTTP status codes as follows:
 In some cases, the frequency of event transmission on an Event Stream will be
 very low, making it difficult for an Event Receiver to tell the difference
 between expected behavior and event transmission failure due to a misconfigured
-stream. Event Receivers can request that a verification event be transmitted
-over the Event Stream, allowing the receiver to confirm that the stream is
+stream. Event Receivers can request that a Verification event be transmitted
+over the Event Stream, allowing the Receiver to confirm that the stream is
 configured correctly upon successful receipt of the event. The acknowledgment of
-a Verification Event also confirms to the Event Transmitter that end-to-end
+a Verification event also confirms to the Event Transmitter that end-to-end
```

Comment:
Don't have a big opinion on the capitalization for this one. I just updated for consistency:
-> "Verification event"
-> "SSF event"

Reply:
Since "Verification Event" is a defined term, we should refer to it in title case, i.e. "Verification Event". Whereas since "SSF event" is a generic term, it can be in sentence case.

Reply:
Will update those refs.